examples/persist-filtering/persist_filtering.md

```diff
@@ -0,0 +1,35 @@
+# Persist read - Filtering and sorting
```

Comment:
if we rename the BBE titles as per the previous comments in index json, fix these titles and URL paths too to match with that.